ui.apps/src/main/content/META-INF/vault/filter.xml

```diff
@@ -61,4 +61,30 @@
         <include pattern="/apps/settings/dam/indesign"/>
         <include pattern="/apps/settings/dam/indesign/scripts(/.*)?"/>
     </filter>
+    <filter root="/apps/wcm">
+        <include pattern="/apps/wcm/core"/>
+        <include pattern="/apps/wcm/core/content"/>
+        <include pattern="/apps/wcm/core/content/editor"/>
+        <include pattern="/apps/wcm/core/content/editor/jcr:content"/>
+        <include pattern="/apps/wcm/core/content/editor/jcr:content/content"/>
+        <include pattern="/apps/wcm/core/content/editor/jcr:content/content/items"/>
+        <include pattern="/apps/wcm/core/content/editor/jcr:content/content/items/content"/>
+        <include pattern="/apps/wcm/core/content/editor/jcr:content/content/items/content/header"/>
+        <include pattern="/apps/wcm/core/content/editor/jcr:content/content/items/content/header/items"/>
+        <include pattern="/apps/wcm/core/content/editor/jcr:content/content/items/content/header/items/headerbar"/>
+        <include
+                pattern="/apps/wcm/core/content/editor/jcr:content/content/items/content/header/items/headerbar/items"/>
+        <include
+                pattern="/apps/wcm/core/content/editor/jcr:content/content/items/content/header/items/acs-commons_copy-publish-url-dialog(/.*)"/>
+        <include
+                pattern="/apps/wcm/core/content/editor/jcr:content/content/items/content/header/items/headerbar/items/pageinfopopover"/>
+        <include
+                pattern="/apps/wcm/core/content/editor/jcr:content/content/items/content/header/items/headerbar/items/pageinfopopover/items"/>
+        <include
+                pattern="/apps/wcm/core/content/editor/jcr:content/content/items/content/header/items/headerbar/items/pageinfopopover/items/list"/>
+        <include
+                pattern="/apps/wcm/core/content/editor/jcr:content/content/items/content/header/items/headerbar/items/pageinfopopover/items/list/items"/>
+        <include
+                pattern="/apps/wcm/core/content/editor/jcr:content/content/items/content/header/items/headerbar/items/pageinfopopover/items/list/items/acs-commons_copy-publish-url(/.*)"/>
+    </filter>
```

Comment:
Ugly but ... realized we dont have a ui.apps.structure package in the project :T